### PR TITLE
Update SelectDB and Apache Doris results

### PR DESCRIPTION
**Pull request by Gabriel39**
Thank you for the contribution!

Please make sure that you've updated `index.html`. In order to do this run `generate-results.sh` from the root directory of ClickBench repository.

### CODE REVIEW

**Comment by Gabriel39** [comment]
@alexey-milovidov PTAL

**Comment by chhetripradeep** [comment]
Hi @Gabriel39 

I will look into it this week. Sorry for the delay and thank you for your patience.

**Comment by Gabriel39** [comment]
@chhetripradeep That's okay. Just keep your pace. Looking forward to your reply soon :)

**Comment by jgq2008303393** [comment]
@chhetripradeep PTAL

**Review by chhetripradeep — APPROVED** [review]
Results are replicable on a c6a.metal instance. Looks good to me 👍

File: doris/results/c6a.metal.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+    "system": "Apache Doris",
+    "date": "2024-04-26",
+    "machine": "c6a.meta, 500gb gp2",
```
[the remark this answers is not in the document]
Comment (Gabriel39) [reply doris/results/c6a.metal.json]: Thanks for your kind remind. Already updated :)

**Comment by Gabriel39** [comment]
Hi @chhetripradeep . I updated index.html by `generate-results.sh` but seems like there's something wrong with `https://benchmark.clickhouse.com/`

**Comment by chhetripradeep** [comment]
Hi @Gabriel39 

Thank you. I was looking at it. Trying to find what caused it.

**Pull request merged**